lua/rocks/operations.lua

```diff
@@ -97,6 +97,9 @@ operations.install = function(name, version, progress_handle)
             if progress_handle then
                 progress_handle:report({ message = message })
             end
+
+            vim.opt.runtimepath:append(vim.fs.joinpath(config.rocks_path, "lib", "luarocks", "rocks-5.1", "*", installed_rock.name:lower()))
```

Comment:
issue: I think putting this in `operations.install` will add plugins to the `rtp` when we upgrade/downgrade a plugin, which I'm not sure we want.
Maybe it should go in `operations.add` instead.

Reply:
I actually specifically added it here because in theory a downgraded plugin could have a vastly different directory structure (or different rtp paths altogether) which won't be refreshed upon changing the plugin. Changing vim.opt forces a refresh internally :)